types.go

```diff
@@ -172,10 +172,13 @@ func (pc PartialCert) ToBytes() []byte {
 // Generally, if highQC.View > highTC.View, there is no need to include highTC in the SyncInfo.
 // However, if highQC.View < highTC.View, we should still include highQC.
 // This can also hold an AggregateQC for Fast-Hotstuff.
+// This can also hola a block.
 type SyncInfo struct {
-	qc    *QuorumCert
-	tc    *TimeoutCert
-	aggQC *AggregateQC
+	qc         *QuorumCert
+	tc         *TimeoutCert
+	aggQC      *AggregateQC
+	block      *Block
+	timoutView View
```

Comment:
`s/timoutView/timeoutView/`

Reply:
fixed in [7813a4e](https://github.com/relab/hotstuff/pull/78/commits/7813a4eb48467f0f2b55986620b6b5e0ee3abe60).